packages/vcaml/vcaml.v0.13.0/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "opensource@janestreet.com"
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+homepage: "https://github.com/janestreet/vcaml"
+bug-reports: "https://github.com/janestreet/vcaml/issues"
+dev-repo: "git+https://github.com/janestreet/vcaml.git"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "async" {>= "v0.13" & < "v0.14"}
+  "async_extra" {>= "v0.13" & < "v0.14"}
+  "base" {>= "v0.13" & < "v0.14"}
+  "core" {>= "v0.13" & < "v0.14"}
+  "core_kernel" {>= "v0.13" & < "v0.14"}
+  "ppx_jane" {>= "v0.13" & < "v0.14"}
+  "angstrom"
+  "angstrom-async"
+  "dune"           {>= "2.0.0"}
```

Comment:
```suggestion
  "dune"           {>= "1.5"}
```
According to the `dune-project` only at least dune 1.5 is needed.
It's not a blocker for to merge this but just if you want to fix this in the future

Reply:
This is indeed an inconsistency in our export process,
thanks for pointing it out.

We do want to depend on dune 2.0+ for configurator-related
issues, so it would make sense to bump the version of the
dune language. (The fix is in general not as simple as a mere
bump, because of deprecations.)